### PR TITLE
Include reference to hearing summary in swagger docs

### PR DESCRIPTION
**Pull request by chris-groves**
## What

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=454&projectKey=LASB&modal=detail&selectedIssue=LASB-587

Add reference to hearing summaries in prosecution cases request spec. This indicates this data is available in v2 of the API.

## Checklist

Before you ask people to review this PR:

- [ ] Tests and linters should be passing
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by arthurashman — DISMISSED** [review]

**Review by arthurashman — APPROVED** [review]

File: swagger/v2/prosecution_case.json [hunk 1]
```diff
@@ -97,7 +97,10 @@
               },
               {
                 "$ref": "offence.json#/definitions/resource"
-              }
+              },
+              {
+                "$ref": "hearing_summary.json#/definitions/resource"
+              },
```
Comment (arthurashman) [inline swagger/v2/prosecution_case.json:103]: Delete this trailing comma for json to work on SwaggerUI

File: swagger/v1/prosecution_case.json [hunk 2]
```diff
@@ -97,6 +97,9 @@
               },
               {
                 "$ref": "offence.json#/definitions/resource"
+              },
+              {
+                "$ref": "hearing_summary.json#/definitions/resource"
```
Comment (arthurashman) [inline swagger/v1/prosecution_case.json:102]: I thought that this wasn't available in v1?

File: swagger/v2/swagger.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ info:
     CDA exposes API endpoints to the LAA's View Court Data and
                           MAAT/MLRA applications and the HMCTS Common Platform in order to
                           exchange criminal court data between the two organisations.
-  version: v2
+  version: v1
```
Comment (arthurashman) [inline swagger/v2/swagger.yaml:9]: Is this intentional? The file is v2/swagger

File: spec/requests/api/internal/v2/representation_orders_request_spec.rb [hunk 4]
```diff
@@ -134,7 +134,7 @@
         response("401", "Unauthorized") do
           let(:Authorization) { nil }
 
-          parameter "$ref" => "#/components/parameters/transaction_id_header"
+          parameter "ref" => "#/components/parameters/transaction_id_header"
```
Comment (laurentqro) [inline spec/requests/api/internal/v2/representation_orders_request_spec.rb:137]: why are we doing this?

File: swagger/v1/prosecution_case.json (hunk 2 above)
Comment (laurentqro) [reply swagger/v1/prosecution_case.json:102]: It is @arthurashman

Comment (laurentqro) [reply swagger/v1/prosecution_case.json:102]: It's hearings that aren't.

Comment (arthurashman) [reply swagger/v1/prosecution_case.json:102]: 👍

**Pull request closed without merging**